fingerprint/fingerprint_test.go

```diff
@@ -39,6 +39,11 @@ func TestFingerprint(t *testing.T) {
 			"A999 B749 8D1A 8DC4 73E5  3C92 309F 635D AD1B 5517",
 			nil, // good fingerprint: no error
 		},
+		{
```

Comment:
Nice!! it's valid for `OPENPGP4FPR` to be lowercase, so we need a test case for this too

Reply:
👍